packages/backend/src/routes/account.js

```diff
@@ -27,9 +27,8 @@ router.post('/create-account/submit', (req, res, next) => {
 });
 
 router.post('/update/submit', (req, res, next) => {
-  if (!req.session.success) {
-    // res.send('not logged in');
-    res.status(status.IM_A_TEAPOT).json();
+  if (req.session.userId == null) {
```

Comment:
Again, the preferred method of null checking in javascript is using the ! operator. This catches: null, undefined, false, 0, and empty strings.